config/locales/views.fr.yml

```
@@ -484,7 +484,7 @@ fr:
     show:
       title: Visite de la société « %{company} »
     unarchive:
-      subjet_unachived: Le sujet n’est plus en échec
+      subjet_unachived: Le besoin n’est plus archivé
```

```suggestion
      subject_unarchived: Besoin désarchivé
```

Oh, attention, la clé s’appellait `subjet_unachived` au lieu de `subject_unarchived`. Deux typos pour le prix d’une.

---

ah oui je n'avais pas vue